content/docs/guides/contribute/contribute-to-code/share-your-changes.fr.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
`share-your-changes` should be `share-changes`

Reply:
Done!